package.json

```diff
@@ -32,7 +32,7 @@
   },
   "homepage": "https://github.com/scality/backbeat#readme",
   "dependencies": {
-    "arsenal": "scality/Arsenal#06dfdd9",
+    "arsenal": "scality/Arsenal#bugfix/ZENKO-1024/add-global-counters",
```

Comment:
just a reminder to update this. I ran tests locally and the new pending tests were passing